apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/audit/ApolloAuditConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.adminservice.audit;
+
+import com.ctrip.framework.apollo.audit.spi.ApolloAuditSpanService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ApolloAuditConfiguration {
```

Comment:
Can we move ApolloAuditSpanService and ApolloAuditConfiguration  to audit-log module?

The logic in portal or admin service are equal? i.e always add some information to RequestContextHolder.getRequestAttributes(); if those information don't exist.